core/src/main/java/org/apache/accumulo/core/file/rfile/bcfile/Compression.java

```diff
@@ -219,7 +220,21 @@ public OutputStream createCompressionStream(OutputStream downStream, Compressor
 
     GZ(COMPRESSION_GZ) {
 
-      private transient DefaultCodec codec = null;
+      private static final String USE_QAT_PROPERTY = "use.qat";
```

Comment:
`use.qat` is succinct, but it might be better to have a more explicit name in the `accumulo.` namespace because this is for affecting Accumulo behavior, something like:

```suggestion
      private static final String USE_QAT_PROPERTY = "accumulo.rfile.compression.quickassist.enabled";
```